pkg/finder/podresourcefinder-internal_test.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package finder
+
+import (
+  // "regexp"
+	"testing"
+
+  	"k8s.io/api/core/v1"
+	. "github.com/smartystreets/goconvey/convey"
+  // "sigs.k8s.io/node-feature-discovery/pkg/podres"
+  // "k8s.io/kubernetes/pkg/kubelet/apis/podresources"
+)
+
+
+
+func TestMakeCPUResourceInfo(t *testing.T) {
+	Convey("Testing makeCPUResourceInfo ", t, func() {
+		Convey(" Empty list of CPUs", func() {
+      cpus := []int64{}
+			resInfo := makeCPUResourceInfo(cpus)
+			Convey("There should be no error", func() {
+        So(len(resInfo), ShouldEqual, 1)
```

Comment:
How do we know the answer is 1?

Reply:
The cpus is an empty list

Reply:
Even if the list of CPUs is empty is should create ResourceInfo instance, inside that the Data would be empty